nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkActionDescriptor.java

```diff
@@ -45,6 +45,9 @@
     @XNode("@httpEnabled")
     public Boolean httpEnabled = Boolean.FALSE;
 
+    @XNode("@sequentialCommands")
+    public Boolean sequentialCommands = Boolean.FALSE;
```

Comment:
Java spliterators use the term "ordered" for that, I think it's better. What do you think?

Reply:
the pb is that "ordered" doesn't imply "no overlap"